src/Components/Books.js

```diff
@@ -0,0 +1,22 @@
+import React from 'react';
+import PropTypes from 'prop-types';
+
+export default function Books(props) {
+  const { title, author } = props;
+  return (
+    <>
+      <ul className="ul-books">
+        <li className="li-books">
+          <span>{title}</span>
+          <span>{author}</span>
+          <button type="submit">Remove</button>
+        </li>
+      </ul>
+    </>
+  );
+}
```

Comment:
![project](https://raw.githubusercontent.com/microverseinc/curriculum-react-redux/main/bookstore/images/bookstore.png?token=GHSAT0AAAAAABYX6HCMX3YBWSTWOMIUHSFGY6KLFIQ)

- [ ] Kindly see that in your design there is a form to **Add New Book** but you don't have it here :fearful: Kindly add the form to follow the requirements :+1:

**Note: Styling is not required at this point :+1:**